src/MTGCore/Services/MTGService.cs

```diff
@@ -35,6 +35,21 @@ public async Task<Card> GetCardByID(int multiverseID)
             return singleCard;
         }
 
+        public async Task<Card> GetCardByID(string Id)
+        {
+            var result = await _client.GetAsync($"cards/{Id}");
+
+            if (!result.IsSuccessStatusCode)
+                return default;
+
+            var stream = await result.Content.ReadAsStringAsync();
+
+            var singleCard = JsonConvert.DeserializeObject<RootObject>(stream).card;
+
```

Comment:
Would it be worth creating a deserialiser class and injecting it? So if a different API were used in the future which returned XML, a new implementation of the deserialiser could be created and pass in that instead?

Reply:
Also naming convention for 'RootObject' could probably be better but it's a minor thing (I know that was probably something from Json2C#)

Reply:
> Would it be worth creating a deserialiser class and injecting it? So if a different API were used in the future which returned XML, a new implementation of the deserialiser could be created and pass in that instead?

Excellent idea. Will create as issue and add to the TODO list.